packages/main/src/MessageStrip.ts

```diff
@@ -151,10 +69,86 @@ const metadata = {
  * @public
  * @since 0.9.0
  */
+@customElement("ui5-message-strip")
+@languageAware
+/**
+ * Fired when the close button is pressed either with a
+ * click/tap or by using the Enter or Space key.
+ *
+ * @event
+ * @public
+ */
+@event("suggestion-scroll", {
+	close: {},
+})
 class MessageStrip extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines the component type.
+	 * <br><br>
+	 * <b>Note:</b> Available options are <code>"Information"</code>, <code>"Positive"</code>, <code>"Negative"</code>,
+	 * and <code>"Warning"</code>.
+	 *
+	 * @type {sap.ui.webc.main.types.MessageStripDesign}
+	 * @defaultvalue "Information"
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property({
+		type: MessageStripDesign,
+		defaultValue: MessageStripDesign.Information,
+	})
+	design!: MessageStripDesign;
+
+	/**
+	 * Defines whether the MessageStrip will show an icon in the beginning.
+	 * You can directly provide an icon with the <code>icon</code> slot. Otherwise, the default icon for the type will be used.
+	 *
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property({ type: Boolean })
+	hideIcon!: boolean;
+
+	/**
+	 * Defines whether the MessageStrip renders close button.
+	 *
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	hideCloseButton!: boolean;
+
+	/**
+	 * Defines the text of the component.
+	 * <br><br>
+	 * <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+	 *
+	 * @type {Node[]}
+	 * @slot
+	 * @public
+	 * @name sap.ui.webc.main.MessageStrip.prototype.default
+	 */
+
+	/**
+	 * Defines the content to be displayed as graphical element within the component.
+	 * <br><br>
+	 * <b>Note:</b> If no icon is given, the default icon for the component type will be used.
+	 * The SAP-icons font provides numerous options.
+	 * <br><br>
+	 *
+	 * See all the available icons in the <ui5-link target="_blank" href="https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html" class="api-table-content-cell-link">Icon Explorer</ui5-link>.
+	 *
+	 * @type {sap.ui.webc.main.IIcon}
+	 * @slot
+	 * @public
+	 */
+	@slot({ type: HTMLElement })
```

Comment:
I think `{ type: HTMLElement }` can be skipped, it should be the default , just `@slot()` should be enough